addons/core/package.json

```diff
@@ -31,6 +31,7 @@
     "doc:toc": "doctoc README.md"
   },
   "dependencies": {
+    "@hashicorp/design-system-components": "^2.9.0",
```

Comment:
Don't we import this in `rose` instead of `core`?

Reply:
Yep. I must have tab completed to the wrong directory. Thanks for catching that. Fixed.